domains/abdullah.json

```diff
@@ -0,0 +1,11 @@
+
+{
+  "owner": {
+    "username": "adistar964",
+    "email": "adistar964@gmail.com"
+  },
+  "description":"this is for my personal website made.",
+  "record": {
+    "CNAME": "ali-developer.netlify.app"
```

Comment:
We currently do not support Netlify or Vercel at the moment. The current workaround is to use [Railway](https://railway.app?referralCode=jj)

Reply:
is gitub pages okay?

I changed it to github-pages link.

Please accept it.